web/impact/impact/v1/helpers/judge_criterion_helper.py

```diff
@@ -1,25 +1,31 @@
 # MIT License
 # Copyright (c) 2017 MassChallenge, Inc.
 
+from django.db.models import F
 from impact.v1.helpers.criterion_helper import CriterionHelper
 
 
 class JudgeCriterionHelper(CriterionHelper):
-    '''Helper for JudgeCriteria. JudgeCriteria specify a feature which 
+    '''Helper for JudgeCriteria. JudgeCriteria specify a feature which
     all applications are "looking for", for example "All startups should be
     read by at least one female judge".
     JudgeCriteria require one CriterionOptionSpec for each feature sought, for
-    example, "one female judge and one male judge" would require two 
+    example, "one female judge and one male judge" would require two
     OptionSpecs. Weight and count can be set independently for each option.
     '''
 
     def option_for_field(self, field):
         return field
 
     def judge_matches_option(self, judge_data, option):
-        return option == judge_data.get(self.cache_judge_field)
+        return option == judge_data.get(self.cache_key)
 
     def analysis_fields(self):
         return [
             self.judge_field,
         ]
+
+    def analysis_annotate_fields(self):
+        return {
+            self.cache_key: F(self.judge_field),
+            }
```

Comment:
seeing as this function is defined in both the children of this class i.e. `JudgeGenderCriterionHelper` and `JudgeRoleCriterionHelper`, I think this can go